source/examples/time-series-forecasting-with-hpo/training_and_evaluation.ipynb

```diff
@@ -0,0 +1,1777 @@
+{
```

Comment:
<p>These maths expressions are not rendering correctly in the Sphinx docs. We might need to enable <code>dollarmath</code> in our <code>conf.py</code> https://myst-parser.readthedocs.io/en/latest/syntax/optional.html.  </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/rapidsai/deployment/pull/255/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='17'/>

Reply:
I've pushed a commit to resolve this